content/cli/workspaces.md

```diff
@@ -190,6 +191,76 @@ These properties specify the compiler to use as well as various options that aff
 
+#### Global generate options
+
+These properties specify the default generate options for when running the `nest generate` command.
+
+| Property Name       | Property Value Type | Description                                                                                                                                                                                                                              |
+| ------------------- | ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `spec`              | boolean *or* object | If assigned to a boolean it can be specified as either `true` or `false` to respectively enable or disable `spec` generation by default, if not specified otherwise in the cli, or in the specific project `generateOptions`. If assigned to an object, each key represents a schematic name, and the boolean value represents if the default spec generation is enabled / disabled for that specific schematic. |
+
+Below is given an example of global generate options, both with spec being a boolean and as an object
+
+```javascript
+{
+  "generateOptions": {
+    "spec": false
+  },
+  ...
+}
+```
+
+or 
+
+```javascript
+{
+  "generateOptions": {
+    "spec": {
+      "service": false
+    }
+  },
+  ...
+}
+```
+
+> error **Warning** Specifying the `spec` as an object, the key for the generation schematic does currently not support automatic alias handling. This means that specifying a key as for example `service: false` and trying to generate a service via the alias `s`, the spec would still be generated. To make sure both the normal schematic name and the alias works, specify both the normal command name as well as the alias, as seen below.
+>
+>```javascript
+> {
+>   "generateOptions": {
+>     "spec": {
+>       "service": false,
+>       "s": false
+>     }
+>   },
+>   ...
+> }
+> ```
+
+#### Project specific generate options
+
+As well as being able to write global generate options, project specific generate options is also available. The project specific generate options follow the exact same format as the global generate options, but are specified directly on each project.
+
+Project specific generate options overwrite global generate options.
+
+```javascript
+{
+  "projects": {
+    "cats-project": {
+      "generateOptions": {
+        "spec": {
+          "service": false
+        }
+      },
+      ...
+    }
+  },
+  ...
+}
+```
+
+> notice **Notice** Project specific generate options overwrite global generate options, meaning that any options set on the project specific generate options take a high precedence than options set in the global generate options. Directly specifying generate specific options on the CLI take the highest precedence than both project specific options and global generate options.
```

Comment:
"Project specific generate options overwrite global generate options, meaning that any options set on the project specific generate options take a high precedence than options set in the global generate options. Directly specifying generate specific options on the CLI take the highest precedence than both project specific options and global generate options."

->

"The order of precedence for generate options is as follows.  Options specified on the CLI command line take precedence over project-specific options.  Project-specific options override global options."